### ISSUE TITLE
please sign-in again

### ISSUE THREAD

**Issue by humanopi** [minecraft-linux/mcpelauncher-manifest#586]
![Captura de tela de 2021-08-16 10-55-03](https://user-images.githubusercontent.com/77795165/129575149-95ac59b7-6349-4ea3-bebe-64280bd06d1d.png)

how to fix

**Comment by faultydev** [comment]
Have you tried going to the settings and pressing "Sign Out"?

**Comment by Saehing** [comment]
Yes, I had a similar problem, but it was solved by a banal login and logout from my google account

**Comment by faultydev** [comment]
should be closed since it was fixed
